README.md

```diff
@@ -201,3 +201,11 @@ CFEngine 3.15.3 (Enterprise) was successfully installed on 'localhost'
 
 Feel free to open pull requests to expand this documentation, add features or fix problems.
 You can also pick up an existing task or file an issue in [our bug tracker](https://tracker.mender.io/issues/?filter=11711).
+
+## Development
+
+To install `cf-remote` so that it reflects any changes in this source directory use:
```

Comment:
Actually, I'd prefer telling people to use `export PYTHONPATH=.` rather the messing up with their installation of `cf-remote`.

Reply:
@vpodzime I'm not familiar with that. That would pre-pend the current directory while in effect and so `cf-remote` would load modules from there? `unset PYTHONPATH` would then remove that and installed `cf-remote` would work as usual?

Reply:
Yes, exactly. with `PYTHONPATH=.` things like `from cf_remote import ssh` would import `./cf_remote/ssh.py`.